.coderabbit.yaml

```
@@ -0,0 +1,22 @@
+  auto_review:
+    base_branches:
+      - main
+      - coderabbit-cleanup
+      - cleanup/*
+    enabled: true
+
+# ADHD-focused review preferences
+review_preferences:
+  # Limit review comments to avoid overwhelming ADHD users
+  max_comments_per_review: 5
+  # Focus on P0/P1 issues only  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)